.github/pull_request_template.md

```diff
@@ -15,9 +15,11 @@ Don't forget to add tests that cover your changes.
 
 Make sure you've run and fixed any issues with these commands:
 
-- `cargo fmt --all -- --check` to check standard code formatting (`cargo fmt --all` applies these changes)
```

Comment:
I'd prefer to leave all this stuff, as these commands are valid for any shell whereas the toolkit.nu only works in nushell. Also, Windows may still be problematic. For instance, if you're running nu.exe from the cargo install directory (~/.cargo/bin) and you try to cargo install again, you'll get a sharing violation and it will fail. MacOS and Linux don't have that issue.

Reply:
i can add them back and only mention the `toolkit check pr` in a note?

Reply:
should be fixed in 6c96dfe6cc1a19839960608150bb62a71b992916 :+1: